src/docs/modules/ROOT/pages/payloads.adoc

```diff
@@ -112,8 +99,8 @@ the data. The mappings are described in the following table.
 |===
 
 For example, a Property `errorMessage` with effective data type `rdf:langString` and the values
-`"Could not load data"@en` and `"Konnte Daten nicht laden"@de` would be serialized in the JSON
-payload as follows: [source,json]
+`"Could not load data"@en` and `"Konnte Daten nicht laden"@de` would be serialized in the JSON payload as follows: [source,json]
```

Comment:
This is currently broken and could be fixed; I think the `[source,json]` needs to be put in a separate line with a blank line before it:
![image](https://user-images.githubusercontent.com/1199036/159450947-f8435c4c-6dcc-4080-aa71-07fd4fdc6591.png)